### PR TITLE
Merge Benchmark

### PR DESCRIPTION
**Pull request by wuyefeilin**
Merge benchmark;
1 Models: hrnet_w18, deelpabv3p_resnet50_vd
2 add fleet and amp

### CODE REVIEW

File: legacy/pdseg/utils/timer.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 def calculate_eta(remaining_step, speed):
     if remaining_step < 0:
         remaining_step = 0
-    remaining_time = int(remaining_step / speed)
+    remaining_time = int(remaining_step * speed)
```
Comment (michaelowenliu) [inline legacy/pdseg/utils/timer.py:22]: double check the calculation.

File: paddleseg/core/train.py [hunk 2]
```diff
@@ -82,6 +83,7 @@ def train(model,
             The 'types' item is a list of object of paddleseg.models.losses while the 'coef' item is a list of the relevant coefficient.
         keep_checkpoint_max (int, optional): Maximum number of checkpoints to save. Default: 5.
         test_config(dict, optional): Evaluation config.
+        fp16 (bool, optional): Whther to use amp.
```
Comment (michaelowenliu) [inline paddleseg/core/train.py]: Whther 写错了

File: paddleseg/core/train.py [hunk 3]
```diff
@@ -96,14 +98,20 @@ def train(model,
             os.remove(save_dir)
         os.makedirs(save_dir)
 
+    # if nranks > 1:
```
Comment (michaelowenliu) [inline paddleseg/core/train.py]: 注释部分可以删掉

**Review by michaelowenliu — CHANGES_REQUESTED** [review]

File: legacy/pdseg/utils/timer.py (hunk 1 above)
Comment (wuyefeilin) [reply legacy/pdseg/utils/timer.py:22]: it is right.

File: paddleseg/core/train.py (hunk 2 above)
Comment (wuyefeilin) [reply paddleseg/core/train.py]: 已修复

File: paddleseg/core/train.py (hunk 3 above)
Comment (wuyefeilin) [reply paddleseg/core/train.py]: 已修复

**Review by michaelowenliu — DISMISSED** [review]

**Review by nepeplwu — APPROVED** [review]

**Pull request merged**